bokehjs/src/lib/models/glyphs/glyph.ts

```diff
@@ -255,13 +255,9 @@ export abstract class GlyphView extends View {
       visual.update()
     }
 
-    this._set_visuals()
-
     this.glglyph?.set_visuals_changed()
   }
 
-  protected _set_visuals(): void {}
-
```

Comment:
This is undoing the changes I introduced in PR #11297 as they are no longer required.